modules/sdk-core/src/bitgo/tss/eddsa/eddsa.ts

```diff
@@ -210,7 +212,8 @@ export async function offerUserToBitgoRShare(
   vssProof?: string,
   privateShareProof?: string,
   userPublicGpgKey?: string,
-  publicShare?: string
+  publicShare?: string,
+  reqId?: IRequestTracer
```

Comment:
Do we not have to add the `reqId` param to `ecdsa.ts` as well?

Reply:
Yes I think I missed those, will add them now